VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -263,11 +272,27 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     private void updateUIForOrientation(){
-        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
+
+        Fragment fragment = null;
+        int currentItem = pager.getCurrentItem();
+
+        if (currentItem < pagerAdapter.getCount()) {
+            fragment = pagerAdapter.getItem(currentItem);
+        }
+
+        boolean isCurrentFragmentVideoUnit = false;
+        if (fragment != null && fragment instanceof CourseUnitVideoFragment)  {
+            isCurrentFragmentVideoUnit = true;
+        }
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE && isCurrentFragmentVideoUnit) {
+            getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
             setActionBarVisible(false);
             findViewById(R.id.course_unit_nav_bar).setVisibility(View.GONE);
-            pager.setEnabled(false);
+            pager.setEnabled(true);
```

Comment:
We can remove the calls to `setEnabled()` since the pager is always enabled now.